website/content/docs/operations/index.mdx

```diff
@@ -0,0 +1,13 @@
+---
+layout: docs
+page_title: Operating Boundary
+description: |-
+  Operational tasks in Boundary.
+---
+
+# Maintaining and Operating Boundary
+
+Having visibility into various components of Boundary plays an important role in ensuring the health of its controllers and workers. While we intend to offer additional monitoring components in subsequent Boundary releases, Boundary can now export metrics for its controllers and workers which can be collected by Prometheus.  These metrics help you understand the current state of your Boundary deployment, thereby, enabling the diagnosis of deployed controllers and workers to help detect anomalies and mitigate. We will cover the metrics available as well as the naming and labeling standards used for the metrics exported. Additionally, we offer the ability to monitor the health of your Boundary instance with new endpoints. Once you defined a listener in our Boundary's configuration file, the controller health endpoint becomes enabled, and a health response may be configured to begin a shutdown process that's based on a configurable amount of time.
```

Comment:
> Once you defined a listener in our Boundary's configuration file, the controller health endpoint becomes enabled, and a health response may be configured to begin a shutdown process that's based on a configurable amount of time.

It looks like we're missing a **have** in the first part of this sentence? Should `enabled` be `available`? The last part of the sentence implies to me that the health response can itself be configured to begin a shutdown process, but I don't think that's right.